### PR TITLE
Update IMU driver to support multiple IMU's

### PR DESCRIPTION
**Pull request by jpanikulam**

### CODE REVIEW

**Comment by jpanikulam** [comment]
In my defense: the power went out before I finished compiling.

File: embedded/imu_driver/imu_driver.cc [hunk 1]
```diff
@@ -16,10 +16,12 @@
...
 namespace embedded {
 
-const std::string i2c_addr = "/dev/i2c-1";
-bool ImuDriver::initialize() {
-  const char* c_str = i2c_addr.c_str();
+bool ImuDriver::initialize(const std::string& i2c_address) {
+  i2c_address_ = i2c_address;
+  const char* c_str = i2c_address_.c_str();
 
+  // Note: The adafruit driver does not take ownership over this string.
```
Comment (IJDykeman) [inline embedded/imu_driver/imu_driver.cc]: Rust please

Comment (jpanikulam) [reply embedded/imu_driver/imu_driver.cc]: 1e9x yes

**Comment by jpanikulam** [comment]
This diff is now ready and tested on the vehicle.

File: embedded/servo_driver/cfg/servo_cfg2.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-index: 2
```
Comment (jaz-jlh) [inline embedded/servo_driver/cfg/servo_cfg2.yaml]: this change should be in a different PR

File: embedded/imu_driver/imu_message.hh [hunk 3]
```diff
@@ -34,7 +37,8 @@ struct ImuMessage : Message {
           mag_utesla_x,
           mag_utesla_y,
           mag_utesla_z,
-          timestamp);
+          timestamp,
+          imu_id);
```
Comment (jaz-jlh) [inline embedded/imu_driver/imu_message.hh]: it's a little confusing that this is also the i2c address - consider renaming this field to `i2c_address` for consistency?

Comment (jpanikulam) [reply embedded/imu_driver/imu_message.hh]: Was using the i2c address as a proxy for a serial number, I renamed the other

**Comment by jpanikulam** [comment]
@jaz-jlh Please review again
